src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -63,6 +63,46 @@ As a result of these changes, users who are assigned will be sent an invitation
 
 You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
 
+### Configuring organization-level roles (Limited Availability)
+
+<Alert level="note">
+
+Organization role provisioning is in limited availability. If you wish to try this feature out, contact support@sentry.io
+
+</Alert>
+
+You can assign an organization-level role to an okta group
+
+1. Add a new custom attribute to your Okta application profile
+
+a. Navigate your your application settings in Okta
+
+b. Under the "Provisioning" tab, select "Go to Profile Editor"
```

Comment:
```suggestion
b. Under the "Provisioning" tab under the "Attribute Mappings" header, select "Go to Profile Editor". If you do not see this, please configure your app for provisioning to Sentry first.
```

Reply:
<img width="799" alt="Screen Shot 2023-01-06 at 09 50 27" src="https://user-images.githubusercontent.com/70817427/211069734-7782edf9-1794-4213-9a35-5a0ffd5c6e58.png">
can we also mention that you need this checked?

Reply:
So the section for role provisioning lives under general Okta SCIM setup page. The page already mentions to `Enable SCIM` and to enable creating users, so I don't think we need to mention it again here.

Reply:
See https://docs.sentry.io/product/accounts/sso/okta-sso/okta-scim/#sentry-configuration